daphne_worker/src/dap.rs

```diff
@@ -501,126 +662,6 @@ impl<'srv> DapAggregator<DaphneWorkerAuth> for DaphneWorker<'srv> {
         Ok(agg_share)
     }
 
```

Comment:
Reviewer note: All of the validation steps below should now be implemented by `DapReportInitializer`.